interface/src/app/utils.js

```diff
@@ -17,4 +17,24 @@ angular.module('adage.utils', [])
 
 // Number of digits in activity value shown on frontend:
 .constant('ActivityDigits', 5)
+
+// Service of global machine learning model, which is shared by multiple
+// modules.
```

Comment:
I thought of this option. But that means I have to either create a new module in `mlmodel` directory, or put it in `mlmodel/mlComponents.js` or `mlmodel/resources.js`, neither filename seems to be a good fit. On the other hand, I feel that it is a service that is shared by all modules, so I put it in `app/utils.js`.